src/content/en/showcase/2017/matterport2.md

```diff
@@ -0,0 +1,122 @@
+{# wf_vertical: media #}
+
+# Matterport WebVR {: .page-title }
+
+*By Håkon Erichsen, Chief Architect, Matterport*
+
+<img src="/web/showcase/2017/images/matterport/top.jpg" class="attempt-right">
+
+Matterport has <a href="https://matterport.com/gallery/">over 450,000 3D Spaces of buildings and
+locations</a> that users can find embedded in sites all over the web. The engineering team is
+always looking to make the experience of exploring our Spaces even more immersive, so VR is a high
+priority for us. When it became clear that WebVR was close to launching, we knew we had to mobilize
+as soon as possible to add support for it.
+
+## The Battle-Plan
+
+Our WebGL player has a lot of different bells and whistles: spatial annotations, automatic tours,
+multiple viewing modes and settings, and more. So when a team of three engineers had only four
+weeks before WebVR launched, we had to get creative. Instead of simply switching to VR mode in our
+current WebGL player, we decided to load a minimal, special WebVR player when the user initiates VR
+mode, and swap to that.
+
+This way we could cut the experience down to a minimal core experience of walking around inside a
+Space, and **ruthlessly optimize** for performance over features, without impacting our millions of
+non-VR users. We also decided to prioritize support for the Daydream headset and controller in
+order to deliver a high quality experience for one device, rather than compromising the experience
+to accommodate a wider range of devices.
+
+## Switching to VR mode
+
+The easiest way we could think of for someone to enter VR was to detect when someone puts their
+phone in a VR headset while having our WebGL player open. This can be done by listening for the
+`vrdisplayactivate` event, which Chrome triggers when a user puts the phone in the Daydream headset
+and the NFC chip is detected.
+
+However, there is a small, but important difference between `vridisplayactivate` and other
+user-initiated events, e.g. onclick: Since we have to unload our normal WebGL player and load the
+WebVR player once we detect the user wanting to go into VR, we need a second or two of loading.
+Using `onclick` we are able to load this in the background during the controller calibration
+screen.  With `vrdisplayactivate`, though, we do not get to start loading until after the
+calibration, which ends up stalling the main thread when the user has the headset on, causing
+uncomfortable flickering:
+
+<a href="https://www.matterport.com/virtual-reality/webvr/"><img
+src="/web/showcase/2017/images/matterport/timeline.png" class="attemptfg-right"></a>
+
+Until `vrdisplayactivate` fires earlier, using `onclick` and a button provides users with the best
+experience for now.
+
+## Performance
+
+We knew performance would have to be rock solid for the experience to work well, so we started with
+the assumption that we would only implement a bare bones experience: no bells and whistles, just
```

Comment:
'so we started with the assumption that we would only implement a bare bones experience:' 

This is a bit wordy. Maybe:

'so we implemented a bare bones experience:'